.github/ISSUE_TEMPLATE/sip.md

```diff
@@ -1,13 +1,13 @@
 ---
 name: SIP
-about: "Superset Improvement Proposal. See https://github.com/apache/superset/issues/5602 for details. The purpose of a Superset Improvement Proposal (SIP) is to introduce any major change into Apache Superset, such as a major new feature, subsystem, or piece of functionality, or any change that impacts the public interfaces of the project"
+about: "Superset Improvement Proposal. See SIP-0 (https://github.com/apache/superset/issues/5602) for details. The purpose of a Superset Improvement Proposal (SIP) is to introduce any major change into Apache Superset, such as a major new feature, subsystem, or piece of functionality, or any change that impacts the public interfaces of the project"
```

Comment:
```suggestion
about: "Superset Improvement Proposal. See SIP-0 (https://github.com/apache/superset/issues/5602) for details. A SIP introduces any major change into Apache Superset's code or process."
```

Reply:
![image](https://github.com/apache/superset/assets/812905/b14b7adc-d405-4c7d-87a1-ecc1ea2844dd)

The description was too long.... I've shortened it :)